EIPS/eip-4671.md

```diff
@@ -0,0 +1,449 @@
+---
+eip: 4671
+title: Non-Tradable Tokens
+description: A standard interface for non-tradable tokens.
+author: Omar Aflak (@omaraflak)
+contributors: Pol-Malo Le Bris, Marvin Martin (@MarvinMartin24)
+discussions-to: https://github.com/ethereum/EIPs/issues/4672
```

Comment:
This should be a forum thread at https://ethereum-magicians.org/

Reply:
Thank you for the comments!